api/client/create.go

```diff
@@ -24,25 +24,41 @@ import (
 
 // Create is responsible for pushing a ToDD object to the server for eventual storage in whatever database is being used
 // It will send a ToddObject rendered as JSON to the "createobject" method of the ToDD API
-func (capi ClientApi) Create(conf map[string]string, objFile string) {
+func (capi ClientApi) Create(conf map[string]string, yamlFileName string) {
 
-	// If no subarg was provided, do nothing special
-	if objFile == "" {
-		fmt.Println("Please provide definition file")
-		os.Exit(1)
+	// See if anything is being passed via stdin
+	fi, err := os.Stdin.Stat()
```

Comment:
http://stackoverflow.com/a/26567513 looks like it may be a bit more robust way of checking this.